bindings/python/test/test_mongocrypt.py

```diff
@@ -370,21 +372,46 @@ def close(self):
 
 
 class TestMongoCryptCallback(unittest.TestCase):
+    maxDiff = None
 
     @staticmethod
     def mongo_crypt_opts():
         return MongoCryptOptions({
             'aws': {'accessKeyId': 'example', 'secretAccessKey': 'example'},
             'local': {'key': b'\x00'*96}})
 
+    def test_csfle(self):
+        # Test that we can pick up CSFLE automatically
```

Comment:
How does this first case pass when `csfle_path` is not set and csfle is not on the system?

Reply:
Gotcha thanks!